winml/lib/Api.Image/TensorToVideoFrameConverter.cpp

```diff
@@ -30,15 +32,45 @@ class GPUTensorToDX12TextureTelemetryEvent {
         TraceLoggingOpcode(EVENT_TRACE_TYPE_START),
         TraceLoggingHexInt32(tensorDesc.channelType, "Type"),
         TraceLoggingInt64(tensorDesc.sizes[2], "Height"),
-        TraceLoggingInt64(tensorDesc.sizes[3], "Width"));
+        TraceLoggingInt64(tensorDesc.sizes[3], "Width"),
```

Comment:
All these telemetry code looks good to me.

Reply:
thanks!